contents/uix/uix.md

```diff
@@ -40,8 +42,72 @@ Here is a non-exhaustive list of UI design concepts you can choose to dive into
 
 ## UX Design
 
+At it's core, UX Design is about understanding both user and business goals, and tailoring a product that strikes a right balance between both goals within given constraints such as budget or time.
+
 Just like Software Development, UX Design is an iterative process. It involves many stages like researching, planning, testing, and at each stage are many different techniques that you could use. A good place to find these techniques would be the [UX Project Checklist](https://uxchecklist.github.io/) created by Andrea Soverini.
 
 ## Which is more important?
 
 The metaphor of building a house by Clayton Yan (UX Designer at UserTesting) illustrates this aptly. UX is the structure (i.e. the number of rooms, one-storey or two-storey and where the front door leads). Meanwhile, UI is about the visual (i.e. whether the same wallpaper or vases is used). A good UX with bad UI is like an ugly house with nice structure. A good UI with bad UX is like a beautiful house with the front door leading straight to the bathroom. To sum it up, UI and UX are **equally** important in creating an awesome experience for the users. 
+
+## The Process of UX Design
+
+As stated above, UX is a highly iterative process to achieve a fine balance between meeting the needs of the target audience and achieving business goals. Many times the functionality of a product is prioritised over the experience it gives. However this should not be the case when experience is the deciding factor in delighting or frustrating users.
```

Comment:
Here is yet another unsolicited review! Could examples be given here to better illustrate this dilemma and how to deal with this dilemma of functionality vs experience? Would it be content vs simplicity? Are there certain UX trends nowadays? Or would it differ for the platform? (Type of website, Android vs Apple etc)

Reply:
hey @leeyh20,

I am thinking that the incorporating a suitable reply would detract away from the scope of the current section, which focuses on the actual stages of UX design and their deliverables.

Perhaps I can include this section in future addition to the UX page?